krun.py

```diff
@@ -101,109 +103,90 @@ def sanity_check_user_change(platform):
 def create_arg_parser():
     """Create a parser to process command-line options.
     """
-    parser = argparse.ArgumentParser(description='Benchmark, running many fresh processes.')
```

Comment:
Right. We may find that we can't be this black and white, for example when nesting quotes. But we can try if you really feel strongly.

Not sure I want to impose any rules TBH.